src/frontend/styles/components/sidebar.pcss

```diff
@@ -1,6 +1,34 @@
 .docs-sidebar {
   width: 100vw;
-    
+
+  &--animated {
+    .docs-sidebar__content {
+      transition: transform 200ms ease-in-out;
+    }
+
+    .docs-sidebar__slider {
+      transition: left 200ms ease-in-out;
+
+      svg {
+        transition: transform 200ms ease-in-out;
+      }
+    }
+  }
+
+  &--collapsed {
+    .docs-sidebar__content {
+      transform: translateX(-100%);
+    }
+
+    .docs-sidebar__slider {
+      left: 20px;
```

Comment:
it's better to use `translate` instead of `left` since it does not fire the reflow.

Reply:
Fixed